server/middleware/measure.go

```diff
@@ -118,9 +121,16 @@ func (w *wrappedStream) RecvMsg(m interface{}) error {
 		err := w.ServerStream.RecvMsg(m)
 		return err
 	}
+
 	childMeasurement := metrics.NewMeasurement(TigrisStreamSpan, "RecvMsg", metrics.GrpcSpanType, parentMeasurement.GetRequestOkTags())
+	reqMetadata, err := request.GetRequestMetadataFromContext(w.Context())
+	if err != nil {
+		childMeasurement.AddTags(map[string]string{
```

Comment:
This will not work. You need to add the tag https://github.com/tigrisdata/tigris/blob/main/server/metrics/requests.go#L28-L58, otherwise it will be removed at the normalisation phase.

Reply:
thanks added it.